docs/registry.md

```diff
@@ -25,6 +25,35 @@ ENote controller
 
 
 
+<a name="io.dartinc.registry.v1beta1.DocumentRecordingGuidance"></a>
+
+### DocumentRecordingGuidance
+Metadata concerning how a collection of documents should be recorded on chain
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| designated_documents | [DocumentRecordingGuidance.DesignatedDocumentsEntry](#io.dartinc.registry.v1beta1.DocumentRecordingGuidance.DesignatedDocumentsEntry) | repeated | Markers for data |
+
+
+
+
+
+<a name="io.dartinc.registry.v1beta1.DocumentRecordingGuidance.DesignatedDocumentsEntry"></a>
+
+### DocumentRecordingGuidance.DesignatedDocumentsEntry
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [bool](#bool) |  |  |
```

Comment:
Should we allow for something more generic than bool?

Reply:
The only alternative I see as viable is a string, which I am not a fan of as:
- it permits multiplying the number of potential hardcoded value that consumers must agree on behavior for
- it enables undesirable usage of this protobuf to directly store any type and amount of data desired, rather than just storing metadata

I like the idea of keeping it as primitive as possible to limit the ways the protobuf can be used and interpreted. The comment I left above replying to @rromanowski-figure has a couple of examples I envisioned. Let me know if you think I'm missing something here.